spec/schemas/_common.yaml

```diff
@@ -802,8 +802,8 @@ components:
       type: string
     StringifiedEpochTimeUnitSeconds:
       description: |-
-        Some APIs will return values such as numbers also as a string (notably epoch timestamps). This behavior
-        is used to capture this behavior while keeping the semantics of the field type.
+        Certain APIs may return values, including numbers such as epoch timestamps, as strings. This setting captures 
```

Comment:
Is this right, or should it be `|` and not `|-`?

```suggestion
        Certain APIs may return values, including numbers such as epoch timestamps, as strings. This setting captures
```

Reply:
Looks like other multi-line descriptions use `|-`, see line 200.

Reply:
1. Make sure end of lines don't have that extra space.
2. Make sure `|-` folds by adding a space?

Reply:
Can do. Did some additional digging into this and most OpenAPI specs use a single line `|`, even for multiline descriptions. I found this article that describes formatting options for descriptions: https://www.baeldung.com/swagger-format-descriptions#yaml-vs-json-formatting

Reply:
Updated all `|-` to `|`.